oauth2client/flask_util.py

```diff
@@ -331,6 +337,20 @@ def _make_flow(self, return_url=None, **kwargs):
             redirect_uri=url_for('oauth2.callback', _external=True),
             **kw)
 
+        flow_key = self._flow_key.format(csrf_token)
+        session[flow_key] = pickle.dumps(flow)
+
+        return flow
+
+    def _get_flow_for_token(self, csrf_token):
+        flow_pickle = session.get(
+            self._flow_key.format(csrf_token), None)
+
+        if not flow_pickle:
```

Comment:
I think `flow_pickle is None` would more precisely convey your authorial intent here than `not flow_pickle`. If `flow_pickle` is the empty string at this point in the code, would you rather return or would you rather crash?